ui/app/components/app/asset-list-item/asset-list-item.scss

```diff
@@ -7,4 +7,24 @@
     margin-top: 6px;
     font-size: 14px;
   }
+
+  &__warning {
+    flex: 1;
```

Comment:
Nit: I suspect this rule might not do anything? Since the `__warning` component only has a single child.

```suggestion
```

Reply:
flex: 1 is applied to the element itself, the parent is display flex, this makes explicit the sizing rules for the two components (the warning and the icon) where previously it was implicit and causes the icon to be resized to fit a gradually changing content box. by telling the __warning element to take up at most as much space that is not occupied by other flex items, it keeps the icon taking up it's appropriate space and allows the text to fill in the remainder. 
![warning](https://user-images.githubusercontent.com/4448075/83904353-55015280-a725-11ea-938f-3c6d3b51192c.gif)

Reply:
Huh. I did try that, but I must not have tried the correct screen size. Thanks, that makes sense.